settings/nyc-msm/model.yml

```diff
@@ -0,0 +1,63 @@
+model:
+  num_pop: 171990
+  time:
+    num_steps: 87
+    burn_steps: 48
+
+classes:
+  races:
+    white:
+      hispanic: false
+    black:
+      hispanic: false
+    latino:
+      hispanic: true
+  sex_types:
+    MSM:
+      gender: M
+      cis_trans: cis
+      sleeps_with:
+        - MSM
+  bond_types:
+    Main:
+      acts_allowed:
+        - sex
+    Casual:
+      acts_allowed:
+        - sex
+  drug_types:
+    - None
+  
+features:
+  prep: true
+  haart: true
+  die_and_replace: true
+  assort_mix: true
+
+outputs:
+  classes:
+    - races
+
+assort_mix:
+  assort_white:
+    attribute: race
+    agent_value: white
+    partner_values:
+      white: 0.649
```

Comment:
does the assorting work as expected? I haven't thoroughly re-tested that with all the refactors?

Reply:
I'll take a look!

Reply:
It seems to, and it works in the tests.